DangerFile

```diff
@@ -2,9 +2,33 @@ junit.parse "test_output/report.junit"
 junit.show_skipped_tests = true
 junit.report
 
+# Warn when there is a big PR
+warn("Big PR, try to keep changes smaller if you can") if git.lines_of_code > 500
+
+# Stop skipping some manual testing
+warn("Needs testing on a Phone if change is non-trivial") if git.lines_of_code > 50 && !github.pr_title.include?("📱")
+
+# Mainly to encourage writing up some reasoning about the PR, rather than
+# just leaving a title
+if github.pr_body.length < 3
+  warn "Please provide a summary in the Pull Request description"
+end
+
+# Ensure a clean commits history
+if git.commits.any? { |c| c.message =~ /^Merge branch '#{github.branch_for_base}'/ }
```

Comment:
I'd rather not have this one. There are many situations where we would prefer a `merge` to a `rebase`. Traditionally, we've always done merges more than rebases because it's cleaner and it doesn't force-rewrite history.

Reply:
Okay.